src/main/java/careneighbors/hospital/Hospital.java

```diff
@@ -0,0 +1,113 @@
+package careneighbors.hospital;
+
+
+
+import jakarta.persistence.*;
+
+@Entity
+public class Hospital {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(nullable = false)
+    private String companyName;
+
+    @Column(nullable = false)
+    private String address;
+
+    @Column(nullable = false)
+    private String contactNumber;
+
+    @Column(nullable = false)
+    private Integer registrationNumber;
+
+    @Column(nullable = false)
+    private String type;
+
+    @Column(nullable = false)
+    private Integer bedCount;
+
+    @Column(nullable = true)
+    private String website;
+
+    @Column(nullable = true)
+    private String imageUrl;
```

Comment:
병원 사진은 여러장일 수도 있지 않을까요? 어떻게 생각하시나요?
```suggestion
   private List<String> imageUrls;
```